xrpl4j-crypto-parent/xrpl4j-crypto-core/src/main/java/org/xrpl/xrpl4j/crypto/core/keys/Seed.java

```diff
@@ -116,6 +136,7 @@ public Seed(final UnsignedByteArray value) {
    */
   @VisibleForTesting
   Seed(final Seed seed) {
+    Objects.requireNonNull(seed);
```

Comment:
Can we add a unit test for this?

Reply:
This was already covered, though I adjusted the null-checks because checking the internals of a typed-seed isn't necessary because they can never really be null.